docs/contributor_guide/development_process.rst

```diff
@@ -19,9 +19,9 @@ functionality. This includes comments in the code itself, docstrings, and user
 guides. For exceptions to this rule the pull request author should coordinate
 with a maintainer. For changes that fix bugs, add new features, change APIs,
 etc., i.e., for changes that are relevant to developers and/or users please
-also add an entry in CHANGES.md in the section corresponding to the *next*
+also add an entry in :code:`CHANGES.md` in the section corresponding to the *next*
```

Comment:
Didn't changes.md basically migrate to the website in the version installation guide? Maybe I'm misremembering...

Reply:
Ooh you are right, that is a file that doesn't exist! What should be referenced there instead of that file? Is there a different process now?

Reply:
Did you see the version and installation guide? It's also part of the docs/website. There should be a page per released version. That is essentially what CHANGES.md used to be.

Reply:
I have somehow missed that, thanks for pointing it out! The guide also answers several questions I've written down, but I haven't had the change to ask @adrinjalali about :) Would the change here in [e828f19](https://github.com/fairlearn/fairlearn/pull/1372/commits/e828f19f9d3338c8a1700e5b74233a3a630f6447) 
 describe accurately what the process is now? I see no new files open in the repo, but I suppose they are opened once a release is prepared?

Reply:
I think @romanlutz is referring to the fact that now we have a separate rst file (like sklearn) for each release. For instance, `v0.6.0.rst`. So this should refer to those files.

Reply:
@adrinjalali I include a link that links directly to the version guide where these files are aggregated. Should I link to a specific file as an example?